_shared/project/tests/functional/sanity_test.py

```diff
@@ -0,0 +1,2 @@
+def test_it():
+    assert True
```

Comment:
This is another file moved without changing it. But `functional/sanity_test.py` is actually a file that the new app shares with the `pyapp` cookiecutter: it's exactly the same as the `sanity_test.py` that's [currently in the `pypackage` cookiecutter](https://github.com/hypothesis/cookiecutters/blob/ebfeeba9ffd65d4de6e65c0d858afecf6fa0a082/pypackage/%7B%7B%20cookiecutter.slug%20%7D%7D/tests/functional/sanity_test.py#L1-L2) on `main`, which this PR replaces with a symlink. The `pyramid-app` cookiecutter has an [`app_test.py`](https://github.com/hypothesis/cookiecutters/blob/ebfeeba9ffd65d4de6e65c0d858afecf6fa0a082/pyramid-app/%7B%7B%20cookiecutter.slug%20%7D%7D/tests/functional/app_test.py#L1-L11) that makes a test request to the app instead of `sanity_test.py`, but that only applies to Pyramid apps.